bin/Ref_Maker

```diff
@@ -50,6 +50,7 @@ sub main { ##no critic (Subroutines::ProhibitExcessComplexity)
         picard    => 0,
         samtools  => 0,
         smalt     => 0,
+	longranger  => 0,
```

Comment:
To help with reading, could we maintain alphabetical order of this list and re-align arrows =>??